sass/_SmartSnippet.scss

```diff
@@ -1,40 +1,158 @@
 @import 'Variables';
 
+@mixin triangle($size: 20px) {
```

Comment:
That already exists in the project if you import `bourbon/boubon`

Grep for `@include triangle` and you will find other examples.

Those things are generally bad for accessibility by the way (:before content in CSS). 

I think it's fine in this case since it's not meaningful for a screen reader.

Reply:
Sweet! I hadn't noticed bourbon has a triangle mixin.

As for the `::before` content, I'd usually avoid it, especially if it displayed text. In this case though, it's just for styling purpose.